Hedvig/Screens/ProfileScreen/Screens/MyPayment/MyPayment.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  MyInfo.swift
+//  Hedvig
+//
+//  Created by Sam Pettersson on 2019-01-04.
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Apollo
+import Flow
+import Form
+import Presentation
+
+struct MyPayment {
+    let client: ApolloClient
+
+    init(client: ApolloClient = HedvigApolloClient.shared.client!) {
+        self.client = client
+    }
+}
+
+extension MyPayment: Presentable {
+    func materialize() -> (UIViewController, Disposable) {
+        let bag = DisposeBag()
+
+        let viewController = UIViewController()
+        viewController.title = ""
+
+        if #available(iOS 11.0, *) {
+            viewController.navigationItem.largeTitleDisplayMode = .never
+        }
+
+        let form = FormView()
+
+        bag += viewController.install(form) { scrollView in
+            bag += scrollView.chainAllControlResponders(shouldLoop: false, returnKey: .next)
```

Comment:
We can remove this I think, we will never have any inputs on this screen so it's not needed.